pkg/reconciler/topic/resources/labels.go

```diff
@@ -0,0 +1,32 @@
+/*
+Copyright 2019 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package resources
+
+import (
+	"k8s.io/apimachinery/pkg/labels"
+)
+
+func GetLabelSelector(controller, source string) labels.Selector {
+	return labels.SelectorFromSet(GetLabels(controller, source))
+}
+
+func GetLabels(controller, topic string) map[string]string {
+	return map[string]string{
+		"cloud-run-pubsub-topic":      controller,
```

Comment:
Should the label prefix here be `pubsub.cloud.run/` to be consistent with the rest of knative?